core/src/main/java/io/kestra/core/utils/Version.java

```diff
@@ -155,8 +159,8 @@ public static Version getOldest(final Collection<Version> versions) {
     }
 
     private final int majorVersion;
-    private final int minorVersion;
-    private final int incrementalVersion;
+    private final Integer minorVersion;
+    private final Integer incrementalVersion;
```

Comment:
Maybe rename it "patch version" like the official [semver](https://semver.org/) name?